art/api/models.py

```diff
@@ -1,3 +1,11 @@
 from django.db import models
 
-# Create your models here.
+
+class MakeModel(models.Model):
```

Comment:
i'm thinking we rename this to `AssetMake` to avoid confusion with the asset model.